scalardl-test/build.gradle

```diff
@@ -9,10 +9,10 @@ repositories {
 }
 
 dependencies {
-    implementation group: 'com.scalar-labs', name: 'kelpie', version: '1.1.0'
+    implementation group: 'com.scalar-labs', name: 'kelpie', version: '1.2.0'
     implementation group: 'com.scalar-labs', name: 'scalardl-java-client-sdk', version: '3.1.0'
     implementation group: 'com.google.inject', name: 'guice', version: '5.0.1'
-    api group: 'com.scalar-labs', name: 'scalardb', version: '2.4.1'
+    api group: 'com.scalar-labs', name: 'scalardb', version: '3.3.0'
```

Comment:
It's fine since it's a non-critical kelpie-test repo, but ideally, this kind of upgrade should be done in another PR.

Reply:
I am very sorry. I will be careful. :bow: